packages/mongo-nav/src/user-menu/UserMenu.spec.tsx

```diff
@@ -135,15 +129,15 @@ describe('packages/mongo-nav/user-menu', () => {
   });
 
   test('renders the account link as a disabled button when set to the empty string', () => {
-    const { getByText, getByTestId } = renderUserMenu({
+    const { getByTestId } = renderUserMenu({
       activePlatform: 'account',
     });
     const trigger = getByTestId('user-menu-trigger');
     fireEvent.click(trigger);
-    const accountButton = getByText('Manage your MongoDB Account')
-      .parentElement as HTMLButtonElement;
+    const accountButton = getByTestId('user-menu-account-button');
+
     expect(accountButton.tagName.toLowerCase()).toBe('button');
-    expect(accountButton.disabled).toBe(true);
+    expect((accountButton as HTMLButtonElement).disabled).toBe(true);
```

Comment:
Why are we typecasting here? It's probably desirable to do something like this if we don't know that accountButton will be defined or contain the disabled attribute:

```js
expect(accountButton?.disabled).toBe(true)
```

Reply:
We're typecasting bc accountButton is inferred as HTMLElement not HTMLButtonElement and HTMLElement doesn't have disabled as an attribute

Reply:
Could we type the `accountButton` const instead of typecasting here?

Reply:
yes that works for sure